practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>Should not be capital letters after each ":".</p><p> </p><p>"Use known user-movie node feature interactions to train a model" -&gt; aren't we in particular actually using the existing edge features to train the model? I'm not sure what you mean by "node feature interactions"</p><p> </p><p>Rather than "unseen edges", perhaps rather "missing edges".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>May be it is the way written, but we are actually using both. Of course we are using the existing edge features but we also incorporate node features in the MPNN. Node features here are created from the data. Updated the text in the next commit.</p>

Reply:
<p>awesome, it's much more clear now</p>